charter/index.md

```diff
@@ -0,0 +1,228 @@
+unless and until new information becomes available
+or unless reopened at the discretion of the Chair(s).
+
+The Chair(s) may appoint individuals to be the
+Editors of the Community Group's Deliverables.
+Such Editors are bound by the consensus of the Community Group
+as determined by the Chair(s),
+and being an Editor does not imply any independent authority
+over the content of the deliverable.
+Editors may nonetheless be allowed by the Chair(s)
+to make non controversial edits
+even in the absence of an explicit group decision,
+or in order to propose some text to be reviewed by the Community Group.
+Even then, if questions or disagreements arise,
+the final determination of consensus remains with the Chair(s).
+
+## Contribution Mechanics
+
+Community Group participants agree to make contributions
+in the GitHub repo
+for the project that they are interested in.
+This may be done by submitting a pull request,
+by raising an issue,
+or by adding a comment to an existing issue.
+The Community Group mailing list must not be used
+for discussing details of specific projects.
+
+Documents maintained by the Community Group
+must use the W3C Software and Document License.
+
+All GitHub repositories attached to the Community Group
+must contain a copy of the CONTRIBUTING and LICENSE files.
+
+## Chair Selection
+
+Participants in this group choose their Chair(s)
+and can replace their Chair(s)
+at any time using whatever means they prefer.
+
+However, if 5 participants
+—no two from the same organization—
+call for an election,
+the group must use the following process
+to replace any current Chair(s) with a new Chair,
+consulting the Community Development Lead
+on election operations (e.g., voting infrastructure and using [RFC 2777](https://tools.ietf.org/html/rfc2777)).
+
+1. **Participants announce their candidacies**.
+Participants have 14 days to announce their candidacies,
+but this period ends as soon as all participants
+have announced their intentions.
+If there is only one candidate,
+that person becomes the Chair.
+If there are two or more candidates,
+there is a vote.
+Otherwise, nothing changes.
+2. **Participants vote**.
+Participants have 21 days to vote for a single candidate,
+but this period ends as soon as all participants have voted.
+The individual who receives the most votes
+—no two from the same organization—
+is elected chair.
+In case of a tie,
+[RFC 2777](https://tools.ietf.org/html/rfc2777) is used to break the tie.
+An elected Chair may appoint co-Chairs.
+
+Participants dissatisfied with the outcome of an election
+may ask the Community Development Lead to intervene.
+The Community Development Lead,
+after evaluating the election,
+may take any action including no action.
+
+## Amendments to this Charter
+
+This Charter can be amended by the Chair(s)
+with consultation of the Community Group,
+if the change is agreed to by W3C’s Community Development Lead
+(Community Development Lead is described in the CG Process).
```

Comment:
We can probably drop this parenthesized note, as the Community Development Lead is also mentioned multiple times elsewhere without clarification… Spell out the “CG Process” to “Community and Business Group Process” and link to the Community and Business Group Process, if this parenthesized note is considered necessary.

Reply:
I would be okay with dropping this line. Anybody else?

Reply:
Yes, drop it.